pkg/geth/receipt_to_core_receipt_test.go

```diff
@@ -18,8 +16,8 @@ var _ = Describe("Conversion of GethReceipt to core.Receipt", func() {
 		receipt := types.Receipt{
 			Bloom:             types.BytesToBloom(hexutil.MustDecode("0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000")),
 			ContractAddress:   common.Address{},
-			CumulativeGasUsed: big.NewInt(21000),
-			GasUsed:           big.NewInt(21000),
+			CumulativeGasUsed: uint64(21000),
+			GasUsed:           uint64(21000),
```

Comment:
I'd be tempted to use different vals for cumulative gas used vs gas used so that we don't accidentally write tests that give false positives (e.g. `Expect(expectedGasUsed).To(Equal(actualCumulativeGasUsed))`)